include/nbl/core/xxHash256.h

```diff
@@ -132,7 +133,92 @@ inline void XXHash_256(const void* input, size_t len, uint64_t* out)
 	out[3] += v4;
 }
 
+constexpr std::array<uint64_t, 4> XXHash_256(const uint8_t* input, const size_t len)
+{
```

Comment:
1. missing namesapce
2. implement the non-constexpr in terms of this one

Reply:
resolved in following commit